webapp/tests/forms/steps/test_steuerlotse_step.py

```diff
@@ -0,0 +1,469 @@
+        title = "The Steuerlotse I"
+        intro = "Not so long ago...."
+        correct_endpoint = "lotse"
+        prev_step = MockStartStep
+        next_step = MockFormStep
+
+        with app.app_context() and app.test_request_context():
+            steuerlotse_step = SteuerlotseStep(title=title, intro=intro, endpoint=correct_endpoint,
+                                               prev_step=prev_step,
+                                               next_step=next_step)
+            steuerlotse_step.name = "This is the one"
+            steuerlotse_step._pre_handle(None)
+
+            expected_render_info = RenderInfo(step_title=title, step_intro=intro, form=None,
+                                              prev_url=url_for(endpoint=correct_endpoint,
+                                                               step=prev_step.name,
+                                                               link_overview=steuerlotse_step.has_link_overview),
+                                              next_url=url_for(endpoint=correct_endpoint,
+                                                               step=next_step.name,
+                                                               link_overview=steuerlotse_step.has_link_overview),
+                                              submit_url=url_for(endpoint=correct_endpoint,
+                                                                 step=steuerlotse_step.name,
+                                                                 link_overview=steuerlotse_step.has_link_overview),
+                                              overview_url=None)
+
+            self.assertEqual(expected_render_info, steuerlotse_step.render_info)
+
+    def test_if_overview_url_set_then_render_info_is_set_correctly(self):
+        title = "The Steuerlotse I"
+        intro = "Not so long ago...."
+        correct_endpoint = "lotse"
+        overview_url = "Everything the light touches..."
+
+        with app.app_context() and app.test_request_context():
+            steuerlotse_step = SteuerlotseStep(title=title, intro=intro, endpoint=correct_endpoint)
+            steuerlotse_step.name = "This is the one"
+            steuerlotse_step._pre_handle(None)
+
+            steuerlotse_step.render_info.overview_url = overview_url
+
+            expected_render_info = RenderInfo(step_title=title, step_intro=intro, form=None, prev_url=None,
+                                              next_url=None, submit_url=url_for(endpoint=correct_endpoint,
+                                                                                step=steuerlotse_step.name,
+                                                                                link_overview=steuerlotse_step.has_link_overview),
+                                              overview_url=overview_url)
+
+            self.assertEqual(expected_render_info, steuerlotse_step.render_info)
```

Comment:
I mean..you are literally setting the overview_url to the correct value here in line 310, soooo how it this testing that it's set correctly?

Reply:
It's testing everything else, that's why the test is called 'test_if_overview_url_set_then_render_info_is_set_correctly'. However, it is confusing. We could also just set the overview_step in the constructor.